common/src/main/res/values/strings.xml

```diff
@@ -34,6 +34,7 @@
     <string name="network_unavailable_return">Return to Wallet</string>
     <string name="button_continue">Continue</string>
     <string name="button_understand">I understand</string>
+    <string name="button_okay">Okay</string>
```

Comment:
Sometimes I wonder why "Okay" keeps showing up in our text rather than "OK", but this is unrelated to this story.  It is based on the designs.